ts/src/header-validator/validate-json.ts

```diff
@@ -1361,6 +1418,44 @@ function eventTriggerData(
   )
 }
 
+function attributionScopeLimit(ctx: Context, j: Json): Maybe<number> {
+  return number(ctx, j)
+    .filter((n) => isInteger(ctx, n))
+    .filter((n) => isInRange(ctx, n, 1, UINT32_MAX))
+}
+
+function maxEventStates(ctx: Context, j: Json): Maybe<number> {
+  return number(ctx, j)
+    .filter((n) => isInteger(ctx, n))
+    .filter((n) => isInRange(ctx, n, 1, UINT32_MAX))
+}
```

Comment:
These function bodies are identical. Can we unify them as `function positiveUint32`?

Reply:
Done.